db/migrate/20190510005427_drop_topics.rb

```diff
@@ -0,0 +1,5 @@
+class DropTopics < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
See my other comment on migrations. Here, I think you'll just want a `def up` and then have  a `def down` that raises an exception `ActiveRecord::IrreversibleMigration`.